application/utils/spreadsheet.py

```
@@ -26,6 +26,18 @@ def findDups(x):
     return {val for val in x if (val in seen or seen.add(val))}
 
 
+def _records_from_worksheet_values(rows: List[List[Any]]) -> List[Dict[str, Any]]:
+    """Build row dicts from raw gspread values without numeric coercion."""
+    if not rows:
+        return []
+    headers = rows[0]
+    records: List[Dict[str, Any]] = []
+    for row in rows[1:]:
+        padded = list(row) + [""] * max(0, len(headers) - len(row))
+        records.append(dict(zip(headers, padded[: len(headers)])))
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)